### PR TITLE
[flutter_test] export fake from flutter_test

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Fake has been moved from mockito to test to facilitate NNBD, re-export this from flutter_test

### CODE REVIEW

File: packages/flutter_test/lib/src/test_compat.dart [hunk 1]
```diff
@@ -21,6 +21,9 @@ import 'package:test_api/src/backend/state.dart'; // ignore: implementation_impo
 // ignore: deprecated_member_use
 import 'package:test_api/test_api.dart';
 
+// ignore: deprecated_member_use
+export 'package:test_api/fake.dart';
```
Comment (goderbauer) [inline packages/flutter_test/lib/src/test_compat.dart:25]: Should we add `show Fake` here to make it super-explicit what we are exporting?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter_test/lib/src/test_compat.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_test/lib/src/test_compat.dart]: Done

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by natebosch — APPROVED** [review]

**Pull request merged**